app/assets/src/components/SampleUpload.jsx

```diff
@@ -53,6 +53,7 @@ class SampleUpload extends React.Component {
       jobQueue: this.sample ? this.sample.job_queue : "",
       memory: this.sample ? this.sample.sample_memory : "",
       branch: this.sample ? this.sample.pipeline_branch : "",
+      alignmentConfigName: this.sample ? this.sample.alignment_config_name : "",
```

Comment:
optional but theoretically this hash could be more cleanly constructed like:

```
if (this.sample) {
  ...
}
```

Reply:
totally. but i think we will refactor on the whole thing pretty soon ~